### DISCUSSION TITLE
tidy packages in project

### DISCUSSION THREAD

**Discussion by gosunuts** [aergoio/aergo#289] [Ideas]
## Suggestion
Currently, there are many packages and files that are unused or unclassified.
Tidy packages and files will make it easier to understand aergo project.

## plan to change ( on progress )
- [x]  remove .travis.yml ( replaced to github actions )
- [ ]  remove doc.go ( duplicated in docs package )
- [x]  remove Dockerfile, Dockerfile.polaris, Dockerfile.tools ( duplicated in Docker package )
- [ ]  remove hooks
- [x]  move message to types/message
- [x]  move examples, tools to docs/examples, docs/tools
- [x]  move polaris to cmd/polaris
- [ ]  move contract/enterprise, contract/name, contract/system to governance/enterprise, governance/name, governance/system

**Comment by kroggen** [comment]
OK good, but some of the last topic can cause conflicts with existing open PRs

The above can be done after the V4 is released, and most PRs are merged.

The priority now must be security issues and the release of V4. This includes the [things that must be done before the release](https://github.com/aergoio/aergo/discussions/277#discussioncomment-7291160)

**Reply by gosunuts** [reply]
ok.
